include/lbann/callbacks/callback_ltfb.hpp

```diff
@@ -50,8 +53,15 @@ class lbann_callback_ltfb : public lbann_callback {
 
   /** Constructor.
    *  @param round_size The number of minibatches in each round.
+   *  @param metric_mode  The expectation for a good tournament metric: increasing/descreasing 
+   *  @todo pair metric_mode with eval_metric
+   *  @param eval_metric Tournament evaluation metrics
+   *  @param selected_weights set of weights to exchange
    */
-  lbann_callback_ltfb(int round_size, lbann_summary* summarizer = nullptr);
+  lbann_callback_ltfb(int round_size, std::string metric_mode,
+                      std::unordered_set<std::string> eval_metrics,
```

Comment:
Does LTFB make sense with multiple metrics?

Reply:
Not at the moment, but there is a future possibility for something like mean/max/min of multiple metrics

Reply:
Would be simpler to make the user manually compute the mean/max/min? We already have layers that can do these operations.